x-pack/plugins/endpoint/server/routes/index.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IRouter } from 'kibana/server';
+
+export function addRoutes(router: IRouter) {
+  router.get(
+    {
+      path: '/api/endpoint/hello-world',
+      validate: false,
+      options: {
+        tags: ['access:resolver'],
+      },
+    },
+    async function greetingIndex(...passedArgs) {
+      const [, , response] = passedArgs;
```

Comment:
see above comment about destructuring

Reply:
oddly it'll complain about dup bindings if you do `function (_, _, response) {` so you'd need something like `function(_1, _2, response) {`. /shrug

Reply:
`function(_context, _request, _response)`, but yeah 😬

Reply:
At any rate. i'm not 100% on changing the variable name to trick static analysis. A bunch of JS devs (including many at elastic) are already/still using `_` prefix to mean 'privateish'. They also use `$` suffix to indicate 'antipattern'. I think these patterns aren't always clear and they don't have long lasting meaning. E.g. tslint officially denounced using `I` prefix for interfaces (i guess to be inline w/ MS) and I've heard kibananas talking about adopting it.

Reply:
My understanding of `$` suffix is that it's generally used as a naming convention for observables. That said, I'm pretty sure that underscore prefix is pretty standard across typescript projects, just like it is in ruby.

If there's a big push back on this, I don't care that much, but I do believe that using something like an underscore prefix with whatever name/type signature a parameter would otherwise have gives a better general ergonomic understanding of what a function has access to (that and it has support in the typescript compiler itself).

Reply:
tsserver supports autofixing w/ `_` prefix so i'm going w/ that 
![image](https://user-images.githubusercontent.com/35559/70057734-a7618b00-15ab-11ea-8ab4-a6fcfe75f695.png)